es/criteria/documenting.md

```diff
@@ -5,48 +5,48 @@ order: 9
 
 ## Requisitos
 
-* Toda la funcionalidad de la codebase -tanto las políticas como el código fuente- DEBE estar descrita en un lenguaje claramente comprensible para aquellos que entienden el propósito del código.
-* La documentación de la codebase DEBE contener
-  * una descripción de cómo instalar y ejecutar el código fuente,
-  * ejemplos que demuestren la funcionalidad clave.
-* La documentación del código base DEBERÍA de contener:
-  * una descripción de alto nivel que sea claramente comprensible para una amplia audiencia de interesados, como el público en general y los periodistas,
-  * una sección que describa cómo instalar y ejecutar una versión independiente del código fuente, incluyendo, si es necesario, un conjunto de datos de prueba,
-  * ejemplos de todas las funcionalidades.
+* Toda la funcionalidad de la codebase - tanto las políticas como el código fuente - DEBE estar descrita en un lenguaje claramente comprensible para aquellos que entienden el propósito del código.
```

Comment:
Spaces not necessary in Spanish: https://es.wikipedia.org/wiki/Raya_(puntuaci%C3%B3n)

If controversial, could be substituted by parenthesis, too.